magellan-legacy/src/main/java/com/wealthfront/magellan/MultiScreen.java

```diff
@@ -0,0 +1,18 @@
+package com.wealthfront.magellan;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public interface MultiScreen<S extends Screen> {
```

Comment:
I'm not against this at all, but curious: what prompted us to split this out into its own interface?

Reply:
Talked offline. This is fine.